src/Tools/ExternalAccess/Razor/RazorDocumentExcerptServiceWrapper.cs

```diff
@@ -28,6 +28,10 @@ public RazorDocumentExcerptServiceWrapper(IRazorDocumentExcerptServiceImplementa
             };
 
             var result = await _impl.TryExcerptAsync(document, span, razorMode, new RazorClassificationOptionsWrapper(classificationOptions), cancellationToken).ConfigureAwait(false);
+
+            if (result is null)
```

Comment:
`result` is `RazorExcerptResult?`

Would have thought some kind of nullability analysis would have warned about this? ¯\\\_(ツ)_/¯

Reply:
Looked into this a bit. Nullable is definitely enabled, so its not a project/targets issue, but definitely no warnings. Given this code:

<img width="427" alt="image" src="https://user-images.githubusercontent.com/754264/168956731-44b675fa-f548-4ddb-8422-093972f9879e.png">

Build output is:
```
Build succeeded.

D:\Code\alt-roslyn\src\Tools\ExternalAccess\Razor\RazorDocumentExcerptServiceWrapper.cs(35,22): warning CS8629: Nullabl
e value type may be null. [D:\Code\alt-roslyn\src\Tools\ExternalAccess\Razor\Microsoft.CodeAnalysis.ExternalAccess.Razo
r.csproj]
D:\Code\alt-roslyn\src\Tools\ExternalAccess\Razor\RazorDocumentExcerptServiceWrapper.cs(36,22): warning CS8602: Derefer
ence of a possibly null reference. [D:\Code\alt-roslyn\src\Tools\ExternalAccess\Razor\Microsoft.CodeAnalysis.ExternalAc
cess.Razor.csproj]
    2 Warning(s)
    0 Error(s)
```

ie, no warning for line 34.

`_impl.TryExcerptAsync` returns `Task<RazorExcerptResult?>`, and `RazorExcerptResult` is a readonly struct. The interface and return type are all defined in this same project.

Something to do with readonly structs maybe? Or because it's a netstandard2.0 project?